src/plugin-loader.ts

```diff
@@ -1,6 +1,7 @@
 import * as ts from "typescript";
 import { Plugin, PluginLoader, PluginInfo } from "sinap-core";
 import { TypescriptPlugin } from "./plugin";
+// import resolve = require("resolve");
```

Comment:
What is this here for?

Reply:
I originally needed it for resolving modules to locations, but then I realized tsc could do everything I needed.

I still need that library in plugin.ts for resolving modules at run time. We still need more work to support importing from non-node_module locations (since we'll need to run tsc on those it seems), but this is still a major improvement.